packages/admin-web-angular/src/app/@shared/product/categories/forms/basic-info/basic-info-form.component.html

```diff
@@ -1,4 +1,19 @@
 <form class="form-horizontal pt-4 pb-0" [formGroup]="form">
+	<div class="form-group row">
+		<label class="col-sm-2 control-label">{{
+			'LANGUAGE' | translate
+		}}</label>
+		<div class="col-sm-9">
+			<select class="form-control" formControlName="locale">
```

Comment:
@tsvetelina-e-y I think we need to load the list of languages from the server instead of hardcode here (I know we do hardcode in some other places, but let's start to do it better way)